contracts/GroupHealth.sol

```diff
@@ -9,13 +9,28 @@ import "./common/UUPSOwnableUpgradeable.sol";
 import "./interfaces/IManager.sol";
 import "./interfaces/IStakedCelo.sol";
 import "./interfaces/IAccount.sol";
-import "./interfaces/IAllowedStrategy.sol";
+import "./interfaces/ISpecificGroupStrategy.sol";
 
 contract GroupHealth is UUPSOwnableUpgradeable, UsingRegistryUpgradeable {
+    /**
+     * @notice An instance of the StakedCelo contract for the StakedCelo protocol.
+     */
     IStakedCelo public stakedCelo;
+
+    /**
+     * @notice An instance of the Account contract for the StakedCelo protocol.
+     */
     IAccount public account;
-    IAllowedStrategy public allowedStrategyContract;
-    IManager public managerContract;
+
+    /**
+     * @notice An instance of the specificGroupStrategy contract for the StakedCelo protocol.
```

Comment:
By convention, in text we've usually been referring to contracts with capitalized name, i.e. specificGroupStrategy -> SpecificGroupStrategy.